editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -21,6 +21,7 @@ type AnalyticsLogDirection =
 
 export type HelpClickMetadata = Record<string, string>;
 export type SelectedUrlsMetadata = Record<"selectedUrls", string[]>;
+export type BackwardsNaviagtionInitiatorType = "change" | "back";
```

Comment:
I thought this might be an extensible option for capturing very similar user behaviour whilst maintaining what initiated it. I could see this expanding to include returning to a section via a section header link.

Reply:
Makes sense, nice to have an enum for this! Another possible future extension in addition to section would be "override" I think - eg via PropertyInformation component, and maybe other node types in future